packages/react/src/components/createInlineOverlayComponent.tsx

```diff
@@ -172,6 +151,46 @@ export const createInlineOverlayComponent = <PropType, ElementType>(
     static get displayName() {
       return displayName;
     }
+
+    private handleIonMount = () => {
+      /**
+       * Mount the inner component when the
+       * overlay is about to open.
+       *
+       * For ion-popover, this is when `ionMount` is emitted.
```

Comment:
What does only the popover emit this? Related to popover positioning?

Reply:
Correct, we ran into an issue with popover implementations with auto widths: https://github.com/ionic-team/ionic-framework/pull/26274.